packages/experiments/src/components/UnifiedPicker/UnifiedPeoplePicker/examples/UnifiedPeoplePicker.Example.tsx

```diff
@@ -167,13 +167,13 @@ export const UnifiedPeoplePickerExample = (): JSX.Element => {
   } as ISelectedPeopleListProps<IPersonaProps>;
 
   return (
-    <>
+    <div role="list">
```

Comment:
Pretty sure this should be unnecessary now that you have `role=list` on the SelectedItemsList itself?